scripts/benchmark.py

```diff
@@ -59,7 +60,7 @@ def main(
             print(f.read())
         print()
 
-        try:
+        with contextlib.suppress(KeyboardInterrupt):
```

Comment:
This is not what we want!

We want keyboard interrupt to kill the process.

Could you push a PR to address? @k1lgor

Reply:
This might be my bad @AntonOsika 

the previous code wasn't killing the process AFAIU
I mean, we didn't change the behaviour, correct?

Reply:
Just create a PR #374 with the revert @AntonOsika @patillacode

Reply:
My bad actually – it's useful to have this!

I created an issue #394 to follow up on improving further

Reply:
I was not crazy then 😅